PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/SparkBatchRemoteRunState.kt

```diff
@@ -71,7 +72,9 @@ open class SparkBatchRemoteRunState(val serverlessSparkSubmitModel: SparkSubmitM
                         err.printStackTrace(PrintWriter(errWriter))
 
                         val errMessage = Optional.ofNullable(err.message)
-                                .orElse(err.toString()) + "\n stack trace: " + errWriter.buffer.toString()
+                                .orElse(err.toString())
```

Comment:
There are two requests:
1. Should we select `err.getCause().getMessage()`? I suppose the current `err` is wrapped by a `RuntimeException` not the source error.
2. Separate the service/system error, tool error and the user error. We can start from this: If the cause exception instanced from IOException, tag it as a service error. If the cause's call stack has our package name, tag it as a tool error, others as user errors